src/python/pants/backend/project_info/dependencies.py

```diff
@@ -16,7 +17,18 @@
     TransitiveTargetsRequest,
     UnexpandedTargets,
 )
-from pants.option.option_types import BoolOption
+from pants.option.option_types import BoolOption, EnumOption
+
+
+class DependenciesOutputFormat(Enum):
+    """Output format for listing dependencies.
+
+    merged: List all dependencies as a single list of targets.
+    json: List all dependencies as a mapping `{target: [dependencies]}`.
+    """
+
+    merged = "merged"
+    json = "json"
```

Comment:
I get the feeling that "structure" is confused with "format" between these two option values. I think we can keep these about format only, with the "basic" one being `text` and the other `json`. That you get more data in JSON than plain text is not surprising to me. Other tools does the same. c.f. the kubernetes cli tool, you have a nice table output in plain text, but can opt to get json output and then you get all the goodies not presented in the plain text table.

Reply:
Alright, I've renamed `merged` to `text`, this makes more sense to me, too.